<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-1RBF983" failures="0" tests="1" name="com.dbank.main.Runner" time="0.311" errors="1" timestamp="2022-10-11T21:30:26 BST" skipped="0">
  <testcase classname="com.dbank.main.Runner" name="runScenario" time="0.311">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.dbank.stepdefinitions.StepDefinitions.i_GET_the_details_for_NACE(StepDefinitions.java:16)
at âœ½.I GET the details for NACE "398481"(file:///C:/Projects/RestAssuredAutomationFramework/src/main/resources/features/naceListValidation/NaceDetails.feature:5)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- com.dbank.main.Runner -->
